src/api/app/controllers/webui/package_controller.rb

```diff
@@ -786,6 +786,7 @@ def live_build_log
 
     if Project.exists_by_name(params[:project])
       @project = Project.get_by_name(params[:project])
+      @project = @project.name if @project.kind_of?(Project)
```

Comment:
They need to be able to deal with being @package as string in any case or they would crash in remote case.

We could rename the variables though to something link @package_string to make clear.